gtests/net/packetdrill/socket.h

```diff
@@ -47,6 +47,7 @@ enum socket_state_t {
 	SOCKET_ACTIVE_CONNECTING,	/* after connect() call */
 	SOCKET_ACTIVE_SYN_SENT,		/* after sending client's SYN */
 	SOCKET_ACTIVE_SYN_ACKED,	/* after client's SYN is ACKed */
+	SOCKET_ACTIVE_MP_JOIN_SENT,
```

Comment:
detail if you have to modify the commit: may you add a comment here?

Another detail, maybe best to use `SOCKET_ACTIVE_SYN_MP_JOIN_SENT`, no?

Reply:
or even better, ```SOCKET_ACTIVE_MP_JOIN_SYN_SENT```

Reply:
Yes, fine for me!